osmosis/assetlist.json

```diff
@@ -27601,6 +28385,33 @@
         "website": "https://coreumcommunitydao.com/",
         "twitter": "https://x.com/Cor3_Community"
       }
+    },
+    {
+      "description": "Commemorative token dedicated to the old Prussian noble family",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/ba-ba",
+          "exponent": 0
+        },
+        {
+          "denom": "BA-BA",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "address": "osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8",
+      "base": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/ba-ba",
+      "name": "von Baysen-Bażeński",
+      "display": "BA-BA",
+      "symbol": "BA-BA",
```

Comment:
Please remove the dash '-'

Reply:
I presume this is an automated PR and may not see human attention, so I went ahead and made the required change